docs/faq.en-US.md

```diff
@@ -0,0 +1,52 @@
+---
+order: 11
+title: FAQ
+---
+
+Here are the frequently asked questions about Ant Design and Blazor that you should look up before you ask in the community or create a new issue. We also maintain a [FAQ issues label](https://github.com/ant-design-blazor/ant-design-blazor/labels/%F0%9F%8C%9F%20Q&A) for common github issues.
+
+If your question is style-related, please refer to the [list of frequently asked questions about Ant Design React](https://ant.design/docs/react/faq-cn)。
```

Comment:
@ElderJames : Bit late to the party but this link probably should be to English version.